src/components/inputs/toggle-input/toggle-input.styles.js

```diff
@@ -0,0 +1,185 @@
+import { css } from '@emotion/core';
+import vars from '../../../../materials/custom-properties';
+
+const toggleSwitchSizes = {
+  small: {
+    height: '16px',
```

Comment:
Can't we use spacings here?

Reply:
Those are all custom numbers, I found it totally confusing how it was before. In this case, I prefer to have them explicitly readable.